src/menu.js

```diff
@@ -863,14 +869,16 @@ exports.get = function(props = {}) {
 
     appMenu.push(...items.slice(0, 2))
 
-    // Remove original 'Preferences' menu item
+    // Remove original 'Preferences' and 'Quit' menu items
 
     let fileMenu = findMenuItem('file')
     let preferenceItem = fileMenu.submenu.splice(
-      fileMenu.submenu.length - 2,
+      fileMenu.submenu.length - 4,
       2
     )[1]
 
+    fileMenu.submenu.splice(fileMenu.length - 2, 2)[1]
+
```

Comment:
Instead you can write

~~~js
.splice(
  fileMenu.submenu.length - 4,
  4
)[1]
~~~

to remove the four last entries, so there's no need for two separate mutations.

Reply:
I have done so. I am sorry for requiring so much feedback, as I am new to JavaScript and I thought that it would be clearer if all that the code defining `preferenceItem`removed was the 'Preferences' option.